tests/test_segment.py

```diff
@@ -156,12 +156,19 @@ def process_func(signal, sampling_rate):
     path = os.path.join(root, file)
     af.write(path, signal, sampling_rate)
 
-    # empty index
+    # empty index and table
     index = audformat.segmented_index()
     result = segment.process_index(index)
     assert result.empty
     result = segment.process_signal_from_index(signal, sampling_rate, index)
     assert result.empty
+    table = audformat.Table(index)
+    result = segment.process_table(table.get())
+    assert result.index.empty
+
+    # non-table object for process_table()
+    with pytest.raises(ValueError):
+        segment.process_table(index)
```

Comment:
I would recommend to test also for the correct error message:

```suggestion
    error_msg = "table has to be pd.Series or pd.DataFrame"
    with pytest.raises(ValueError, match=error_msg):
        segment.process_table(index)
```